source/operations/concepts.rst

```diff
@@ -149,12 +149,14 @@ For more information on write preference calculation logic, see :ref:`Writing Fi
 Rebalancing data across all pools after an expansion is an expensive operation that requires scanning the entire deployment and moving objects between pools.
 This may take a long time to complete depending on the amount of data to move.
 
-Starting with MinIO Client version RELEASE.2022-11-07T23-47-39Z, you can manually initiate a rebalancing operation across all server pools using :mc:`mc admin rebalance`. 
+MinIO does not recommend manual rebalancing.
+If required, you can manually initiate a rebalancing operation across all server pools using :mc:`mc admin rebalance`. 
```

Comment:
Maybe
```suggestion
If absolutely necessary, you can manually initiate a rebalancing operation across all server pools using :mc:`mc admin rebalance`. 
```

Reply:
Maybe even
```suggestion
For extreme situations, MinIO provides a command to :mc:`manually start a rebalance operation <mc admin rebalance>`. 
```

With this, they have to take the extra effort of clicking through to even find what the command is.

Reply:
By request, leaving text as the less scary version. (See other comments.)